lib/s3_meta_sync/syncer.rb

```diff
@@ -195,10 +195,10 @@ def delete_local_files(local, paths)
     end
 
     def s3
-      @s3 ||= ::Aws::S3::Client.new(
+      @s3 ||= Aws::S3::Client.new(
         access_key_id: @config[:key],
         secret_access_key: @config[:secret],
-        region: @config[:region] || 'us-west-2'
```

Comment:
don't change behavior

Reply:
We agreed to this per an offline discussion 🤔

Reply:
pr docs say "S3 is no longer global" and then here the region is removed ... I'm confused

Reply:
Ohhhh, that's because I fucked up when copying these changes from my initial branch on the master fork 🤦‍♂️ I meant to just remove the fallback of `us-west-2`

Reply:
and why is the fallback getting removed ?

Reply:
that will break users that currently rely on it

Reply:
the default region for S3 is currently us-east-1. From my understanding this default was selected while S3 was global, and is no longer a relevant default.

I mentioned changing the fallback to us-east-1 and you suggested that it was safer for just removing the fallback altogether (unless I misunderstood you).

Reply:
per offline discussion: we've decided to leave the existing default in since it could break existing user's configuration.